### PR TITLE
Create headings for sidebar pages

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
- [x] Blocked by https://github.com/nextcloud/server/pull/36640

Relates to https://github.com/nextcloud/server/issues/35626, fixes "Activity" part

After:

Screenshot from 2023-02-13 15-41-21

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ $(function(){
 			var navigationLink = this.$navigation.find('a[data-navigation=' + filter + ']');
 			navigationLink.parent().addClass('active').attr('aria-current', 'page');
 			window.document.title = navigationLink.text().trim() + ' - ' + this.defaultPageTitle;
-
+			document.getElementById('pageHeadingLevel2').textContent = `${t('active', 'Subsection: ')} ${navigationLink.text().trim()}`;
```
Comment (nickvergessen) [inline js/script.js:62]: String concatenation breaks translations:

https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html#important-notes

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Comment by nickvergessen** [comment]
Where is the section defined. I assume a generic PR in server? How is ensured that it is not added when there is an h1 already.
Why are the words "Section" and "Subsection" added? That feels so wrong and unneeded.
If it would be helpful we could make activity filters have defining a "Activitiy by others" h2 title additional to the link name.
But the current wording really sound like a quirk/hack?🤔

**Review by Pytal — APPROVED** [review]

**Review by artonge — APPROVED** [review]

**Pull request merged**